app/jobs/email_privacy_policy_updates_job.rb

```diff
@@ -0,0 +1,31 @@
+class EmailPrivacyPolicyUpdatesJob < ApplicationJob
+  def perform
+    petitions.find_each do |signature|
+      EmailPrivacyPolicyUpdateJob.perform_later(signature)
+    end
+  end
+
+  private
+
+  def petitions
+    open_petitions.or(closed_petitions).or(rejected_petitions)
```

Comment:
I'm still wondering whether it would make sense to run the clean-up anonymization job first. 

Potentially then we could just send the email to all signatories of petitions that _hadn't_ been anonymised?

I'm also thinking that '6 months ago' when we run the first task will be earlier than '6 months ago' by the time we run the this task, leaving some petitions that haven't been anonymized and also don't get the email sent. Does this make sense?

Reply:
I guess the other way would be to pass a date into the task as a parameter, so we would know that anything after the 1st June (for example) had to be sent an email. But that seems a bit trickier. I think your suggested way works nicely.